lib/classes/Service.js

```diff
@@ -93,6 +93,7 @@ class Service {
         that.plugins = serverlessYml.plugins;
         that.resources = serverlessYml.resources;
         that.functions = serverlessYml.functions;
+        that.runtime = serverlessYml.runtime;
```

Comment:
We could define a couple of provider specific variables that we simply include when they are available? Imho Runtime is something that while it is provider specific is super helpful to have and most providers will have it. So we can expose it if its their and simply not if it isn't? Don't think this would lead to issues for users as you wouldn't rely on runtime when its not there.

General question would be if we want to expose all provider specific variables automatically, but imho thats a more complex question we should go into at a later stage.